dts/bindings/arm/atmel,sam0-device_id.yaml

```diff
@@ -0,0 +1,21 @@
+---
+title: Atmel Serial Number binding
```

Comment:
W would call it Device Id instead of Serial number.

Reply:
Ok.